CHANGELOG.md

```diff
@@ -9,16 +34,16 @@
 
 ### Features
 
-* **apix:** adds scrolling to frame elements ([#526](https://www.github.com/looker-open-source/sdk-codegen/issues/526)) ([d7cd769](https://www.github.com/looker-open-source/sdk-codegen/commit/d7cd76917522c37e2902792405a75b8b9358e92f))
-* **codegen:** Added --versions option to the code generator ([#514](https://www.github.com/looker-open-source/sdk-codegen/issues/514)) ([ee6f3e8](https://www.github.com/looker-open-source/sdk-codegen/commit/ee6f3e8f55e300df1a75c9be89b47f067bc08dee))
-* **codegen:** added -n | --nostreams to code generator options ([#549](https://www.github.com/looker-open-source/sdk-codegen/issues/549)) ([6ead15a](https://www.github.com/looker-open-source/sdk-codegen/commit/6ead15a26093cc108912c8082b7267fb3a0b76b3))
-* **codegen:** Looker 21.4 bindings ([#529](https://www.github.com/looker-open-source/sdk-codegen/issues/529)) ([4ecaec9](https://www.github.com/looker-open-source/sdk-codegen/commit/4ecaec93d991b9d82fd3a9ce584ee6ae8810341f))
+* **api-explorer:** adds scrolling to frame elements ([#526](https://www.github.com/looker-open-source/sdk-codegen/issues/526)) ([d7cd769](https://www.github.com/looker-open-source/sdk-codegen/commit/d7cd76917522c37e2902792405a75b8b9358e92f))
+* **sdk-codegen:** Added --versions option to the code generator ([#514](https://www.github.com/looker-open-source/sdk-codegen/issues/514)) ([ee6f3e8](https://www.github.com/looker-open-source/sdk-codegen/commit/ee6f3e8f55e300df1a75c9be89b47f067bc08dee))
+* **sdk-codegen:** added -n | --nostreams to code generator options ([#549](https://www.github.com/looker-open-source/sdk-codegen/issues/549)) ([6ead15a](https://www.github.com/looker-open-source/sdk-codegen/commit/6ead15a26093cc108912c8082b7267fb3a0b76b3))
+* **sdk-codegen:** Looker 21.4 bindings ([#529](https://www.github.com/looker-open-source/sdk-codegen/issues/529)) ([4ecaec9](https://www.github.com/looker-open-source/sdk-codegen/commit/4ecaec93d991b9d82fd3a9ce584ee6ae8810341f))
```

Comment:
were these changes manual?

Reply:
yeah - I'm still tweaking the aggregate changelog format: I'm still not quite sure what the right approach here is - I'm actually starting to think maybe we DO want to include a `(scope)` in all our PRs so that these are derived directly from that. The tradeoff is redundancy in the per-package changelog but maybe a little redundancy never hurt anyone?

Reply:
Redundancy is fine IMO. Better than lost info.